e2e/testdata/cloning/clone-pod-template.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
 spec:
   containers:
     - name: ubuntu
-      image: quay.io/cybozu/ubuntu:20.04
-      command: ["/usr/local/bin/pause"]
+      image: ubuntu:20.04
+      command: ["/usr/bin/sleep", "infinity"]
```

Comment:
Can we use the pause image?

Reply:
Cloning test cases uses some commands in this container.

https://github.com/topolvm/topolvm/blob/main/e2e/clone_test.go

Reply:
Does it make sense to do something like `pause` command?

```bash
      image: ubuntu:20.04
      command:
        - bash
        - -c
        - |
          sleep inf &
          trap "kill -SIGTERM $!" SIGTERM
          wait $!
          exit
```

Or improving e2e test not to use commands that are not in the pause container looks good to me.

Reply:
I think it works.

Reply:
@daichimukai @toshipp I modified to above commands.

Reply:
Thank you.